docs/ros/tutorials/cheat_sheet.mdx

```diff
@@ -245,6 +245,18 @@ Supported licenses include:
 | `src/`                     | C++ source files |
 | `action/`, 'msg/`, 'srv/`  | ROS action, message, and service definitions |
 
+### Releasing packages
```

Comment:
This topic seems advanced for a cheat sheet.  A release repo is needed prior.  Also, I believe the rosdistro maintainer's would not like PRs from packages that haven't passed pre-release tests.

Reply:
This was included in the old Noetic/Melodic cheat sheet, so marketing requested it be added here too.